_data/government.yml

```diff
@@ -13,6 +13,17 @@ websites:
       tfa: No
       twitter: canrevagency
 
+    - name: DigiD
+      url: https://www.digid.nl/
+      img: digid.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      phone: Yes
+      doc: https://www.digid.nl/nl/vraag-en-antwoord/hoe-kan-ik-de-sms-controle-toevoegen/
+      exceptions:
+        text: "Software implementation requires use of the DigiD app. It does not support multiple accounts on the same device. To enable the app, the user must already have the SMS option configured."
```

Comment:
Does the app require SMS, or just a linked phone number? For example, could I register a landline and use phone call verification for the app? If it does require SMS, I think we could simplify this to: `SMS required for software 2FA. 1 account per device.` If it only requires a valid phone number, we could simplify to: `SMS or phone call required for 2FA. 1 account per device.`

Reply:
Activating the app (2FA through software, that is) requires you to enter a code once that you receive through SMS or a phone call. So is rephrasing it to the following better or is it still too long: `Software 2FA requires DigiD app. 1 account per device. SMS or phone call required once for activating app.`?

Reply:
Does the phone number used to activate the app need to remain on the account after the app is activated?

Reply:
Yes, that's required. After activating the app you can then choose to log in using the app (by scanning a QR code that's displayed and approving the login request) or by logging in using username, password and a code (from SMS or phone call).

Reply:
In that case, use `Software 2FA requires DigiD app and SMS or phone call. 1 account per device.` since the phone number must remain on the account after activating the app.

Reply:
I have changed the exception text.